lib/pka_dev.h

```diff
@@ -33,7 +33,8 @@
 #define PKA_VFIO_CONTAINER_PATH     "/dev/vfio/vfio"
 #define PKA_VFIO_GROUP_FMT          "/dev/vfio/%d"
 
-#define PKA_DEVFS_RING_DEVICES      "/dev/pka/%d"
+#define PKA_DEVFS_RING_DEVICES_PKA  "/dev/pka/%d"

```

Comment:
I don't think this is a good macro name. I would suggest `PKA_DEVFS_RING_DEVICES_DEPRECATED`. I couldn't make sense of having PKA both as prefix and suffix. This macro is for ring devices file path/name. It should be specific and descriptive of that path (the macro value). I leave it up to you, you may propose a different name; my point is simply that we need to understand from the macro name that we have a path that is deprecated or old and a path that is new. Also please feel free to rename the original macro if needed.